README.md

```diff
@@ -1,27 +1,27 @@
-# Elastic Distributions for OpenTelemetry
+# Elastic Distributions of OpenTelemetry
 
 [OpenTelemetry](https://opentelemetry.io/docs/) is a vendor-neutral observability framework for collecting, processing, and exporting telemetry data.
-The Elastic Distribution for OpenTelemetry Collector (Elastic OTel Collector) is a supported, drop-in replacement distribution of the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) made up of mostly upstream components.
-You can send your telemetry data to Elastic Observability using OpenTelemetry the following ways:
+The Elastic Distribution of OpenTelemetry (EDOT) Collector is a supported, drop-in replacement distribution of the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) made up of mostly upstream components.
```

Comment:
This section goes immediately about the EDOT Collectior making it seem as if the EDOT collector is everything.

See the title is Elastic Distributions of OpenTelemetry which includes collector and language SDKs separately. See this [blog draft](https://docs.google.com/document/d/1ExB_L5_gcQbjyGtr9S_zT7s2trOz2CZGiyW1dHGthe0/edit) for details on the full ecosystem